src/datasources/results/components/query-builders/steps-querybuilder-container/StepsQueryBuilderContainer.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import { InlineField } from '@grafana/ui';
+import { ResultsQueryBuilder } from '../query-results/ResultsQueryBuilder';
+import { StepsQueryBuilder } from '../query-steps/StepsQueryBuilder';
+import { QueryStepsDataSource } from 'datasources/results/query-handlers/query-steps/QueryStepsDataSource';
+import { Workspace } from 'core/types';
+import { enumToOptions } from 'core/utils';
+import { TestMeasurementStatus } from 'datasources/results/types/types';
+import { QueryBuilderProps } from 'smart-webcomponents-react/querybuilder';
+
+type Props = QueryBuilderProps &
+React.HTMLAttributes<Element> & {
+  datasource: QueryStepsDataSource;
+  resultsQuery?: string;
+  stepsQuery?: string;
+  onResultsQueryChange: (query: string) => void;
+  onStepsQueryChange: (query: string) => void;
+  disableStepsQueryBuilder: boolean;
+}
+
+export const StepsQueryBuilderContainer = (
```

Comment:
Could we rename it better:
```suggestion
export const StepsQueryBuilderWrapper = (
```